All Core Devs Meetings/Meeting 121.md

```diff
@@ -0,0 +1,1436 @@
+
+**Marius**
+
+uh I I would just like to react to that I think that's a bad argument that um we would force current clients to change the
+
+architecture by increasing the gas limits I think all current clients are looking into increasing into changing the architecture in in similar ways as eragon does
+
+so the people are already looking at it putting pressure on the teams is just not going to increase uh the the speed in which this is going to be implemented um the other the other small comment I have
+
+that it's currently in my opinion it's not about uh state growth with the current uh gas limit it's about dos I'm not sure if all of you are aware away but aware but
+
+like there are some dos vectors we found some dos vectors uh recently um and um it's pretty hard to uh measure this and so yeah I don't think this this this
+
+parameter is extremely dangerous and it should not be in the hands of people that are not familiar with what's really going on in the network
+
+# EIP status updates
+
+**Tim**
+
+um yeah thanks for sharing uh yeah just because we're already past time you know we can obviously continue this conversation on discord and bring it up on a future call I think we have kind of some you know definitely areas to look at
+
+um I think Pooja you had put a couple EIPs on the agenda uh EIP 2364 and 2464 which are basically the eth 64 and eth 65 protocol EIPs uh as I understand that the issue is like both of those are shipped but the EIPs are still like in draft right
+
+**Pooja**
+
+right so the main issue here is uh EIP-2481 which is for Eth 66. that is uh in the last call actually the last call duration has also passed and we would want to move that to the final status
+
+but the problem is that proposal requires Eth 65 which is 2464 and Eth 65 requires Eth 64 which is two three six four and uh both these proposals are still in draft status so we would want that these two
+
+proposals should move to the final before we could make move EIP 2481 to the final status I'm happy to make the request to request a status change it's just that we wanted to make sure that it
+
+is in knowledge of geth team if anyone from get team wants to volunteer and do that fair enough and if not then we can uh do that and we would need just author's approval
+
+**??**
+
+so thanks for the initiative what I can say is that I think the last time we tried
+to do something like this there was like a huge amount of backlash for some reason because then people came on and you know like wanted to actually see some justification for these EIPs even
+
+though they are like four years old or something I don't really know like this is definitely something that I would like to avoid so I feel like these EIPs they have been you know we don't even
+
+use 864 anymore it's already like past you know it's it's it's basically it's already happened like it we there is really no reason not to move it to the final because
```

Comment:
```suggestion
use Eth 64 anymore it's already like past you know it's basically it's already happened like it we there is really no reason not to move it to the final because
```